### ISSUE TITLE
Crashes on launch

### ISSUE THREAD

**Issue by waughla** [files-community/Files#14510]
### Description

I've installed the classic stable version and the preview, both crash either immediately or a few seconds after launch.

### Steps To Reproduce

1. Install
2. Launch

### Requirements

Fix crash upon launch.

### Files Version

Unsure

### Windows Version

10.0.23619

### Log File

N/A

**Comment by yaira2** [comment]
Is there a log file in the app directory?

**Comment by waughla** [comment]
I'd be happy to check, can you direct me to where that location is?

**Comment by yaira2** [comment]
` To upload the log, access it from Settings -> About -> Open log location; Or
        - Select the `debug.log` file from `%localappdata%\Packages\49306atecsolution.FilesUWP_et10x9a9vyk8t\LocalState`
        - Drag and drop the file to upload as an attachment`

**Comment by waughla** [comment]
That package directory does not exist on my computer.

________________________________

**Comment by yaira2** [comment]
That's for the store version, this is for the preview version `%localappdata%\Packages\FilesPreview_1y0xx7n9077q4\LocalState`.

**Comment by waughla** [comment]
OK I did a reinstall and it's the same crash. Attaching log from preview.

Stable classic install crashes at the same point, right after Creative Cloud.

**Comment by waughla** [comment]
Its crashing at every cloud provider, Adobe, Dropbox, iCloud, OneDrive. Both versions.

I've uninstalled the first three but I cannot touch OneDrive, of which I have three accounts.


From: Kevin Waugh
Sent: Monday, January 22, 2024 12:46 AM
To: files-community/Files ***@***.***>
Subject: RE: [files-community/Files] Crashes on launch (Issue #14510)

OK I did a reinstall and it's the same crash. Attaching log from preview.

Stable classic install crashes at the same point, right after Creative Cloud.
